test/oxo.js

```diff
@@ -0,0 +1,110 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const OXO = artifacts.require('./OXO.sol');
+
+contract('OXO Game', function (accounts) {
+  const reverter = new Reverter(web3);
+  afterEach('revert', reverter.revert);
+
+  const asserts = Asserts(assert);
+  const Player1 = accounts[1];
+  const Player2 = accounts[2];
+  let oxoGame;
+
+  before('setup', () => {
+    return OXO.deployed()
+      .then(instance => (oxoGame = instance))
+      .then(reverter.snapshot);
+  });
+
+  it('should allow to create game', async () => {
+    await oxoGame.createGame({from: Player1, value: 100});
+    var result = await oxoGame.createGame.call({from: Player1, value: 100});
+    assert.equal(result, 1);
+  });
+
```

Comment:
'should throw when create a game with zero bet'

Reply:
Why not, If rabbits play =)